js/core/errors.js

```diff
@@ -246,5 +246,10 @@ export default errorUtils({
     /**
     * @name ErrorsCore.W0017
     */
-    W0017: 'Filter query string exceeds maximum length limit of {0} characters.'
+    W0017: 'Filter query string exceeds maximum length limit of {0} characters.',
+
+    /**
+    * @name ErrorsCore.W0018
+    */
+    W0018: '"mouseleave" event can\'t be used as "hideEvent" when "shading" option is true'
```

Comment:
Let's discuss it with technical writers

Reply:
There is no need to designate a particular event in the warning. We must indicate that we completely ignore this option when shading is used